subsys/net/lib/lwm2m/lwm2m_obj_server.c

```diff
@@ -90,6 +104,55 @@ static int update_trigger_cb(u16_t obj_inst_id)
 #endif
 }
 
+void *binding_pre_write(u16_t obj_inst_id, size_t *data_len)
```

Comment:
Let's add a comment here explaining that binding_pre_write sets up an alternate buffer to use for data validation